server/server.ts

```diff
@@ -31,6 +34,34 @@ app.get('/', (_req: Request, res: Response) => {
     .sendFile(path.resolve(import.meta.dirname, '../client/index.html'));
 });
 
+app.post(
+  '/api',
+  // userQueryController,
+  openaiNLPQuery,
+  googlePlacesApi,
+  openAIRecommendationResponse,
+  //logger?
+  (_req, res) => {
+    res.status(200).json(res.locals.localRecommendation);
+  },
+);
+
+//router to insert userdata into database
+app.post('/insert', insertUserDataMiddleware, (req: Request, res: Response) => {
```

Comment:
I'm not sure this is neccessary, as the only logging we would be doing is during the `POST /api/recommendations` route handling